getting-started/singleframe.rst

```diff
@@ -36,11 +36,11 @@ Reviewing what data will be processed
 =====================================
 
 Pipeline tasks can operate on a single image or iterate over multiple images.
-You can query for the data available to process using the :command:`butler` command line utility.
+You can query for the data IDs of the exposures available to process using the :command:`butler` command line utility.
 
 .. code-block:: bash
 
-   butler query-data-ids SMALL_HSC --collections HSC/RC2/defaults --datasets 'raw'
+   butler query-data-ids SMALL_HSC exposure --collections HSC/RC2/defaults --datasets 'raw'
```

Comment:
I think you need to also add `detector` if you want to match the output shown after this. I think @TallJimbo would prefer it changes to query-datasets but that might be a bigger change to the flow of the tutorial than you want to make.

Reply:
Good catch: I've added detector. I'm not going to change it to query-datasets (though that's probably a good idea), since that will require reworking the rest of the text.